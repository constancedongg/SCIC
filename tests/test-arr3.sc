int func main() {
    int x = 5;
    int[] arr = [2 * x, x * x, (10 - 3) * 2, 5];
    int i = 0;
    for (; i < 4; i = i + 1) {
        print(arr[i]);
    }
    return 0;
}