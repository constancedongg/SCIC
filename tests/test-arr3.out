10
25
14
5